oeps/oep-0001.rst

```diff
@@ -45,163 +61,211 @@ OEP Types
+Draft OEPs and revived OEPs that need a new Arbiter (if the original Arbiter is no
+longer available). The team can also be a resource to help/advise the Arbiter with
+the OEP process.
 
-The OEP Author should first attempt to ascertain whether the idea is appropriate 
-for an OEP.  Posting to the `edx-code`_ mailing list is the best way to go about 
-this.
+.. _edX Architecture Team: https://openedx.atlassian.net/wiki/spaces/AC/pages/439353453/Architecture+Team
+
+OEP Workflow
+============
+
+.. contents::
+  :local:
+  :depth: 2
+
+Submitting an OEP
+-----------------
+
+Step 1. Scope your idea
+~~~~~~~~~~~~~~~~~~~~~~~
+
+The OEP process begins with a new idea for Open edX. Unless it is a Best
+Practice, a single OEP should contain a single key proposal or new idea. Small
+enhancements or patches often don't need an OEP and can be injected into the Open
+edX development workflow with a patch submission.
+
+Step 2. Vet your idea
+~~~~~~~~~~~~~~~~~~~~~
+
+OEP Authors should first attempt to ascertain whether the idea is appropriate 
+for an OEP. Posting to the `open-edx-proposals Slack channel`_ is the best way
+to go about this.
 
 Vetting an idea publicly before going as far as writing an OEP is meant to save
-the potential author time. Many ideas have been brought forward for changing
-Open edX that have been rejected for various reasons. Asking the Open edX
-community first if an idea is original helps prevent too much time being spent
-on something that is guaranteed to be rejected based on prior discussions
-(searching the Internet is not always sufficient). It also helps to make
-sure the idea is applicable to the entire community and not just the author.
-Just because an idea sounds good to the author does not mean it will work for
-most Open edX installations.
-
-Initial Submission
-------------------
-
-Once the Author has asked the Open edX community whether an idea has any chance
+the potential authors time. Asking the Open edX community first if an idea is 
+original and appropriate helps prevent wasted effort. It also helps to make
+sure the idea is applicable to the entire community and not just the authors.
+
+.. _open-edx-proposals Slack channel: https://openedx.slack.com/messages/C1L370YTZ/details/
+
+Step 3. Create PR for "Draft" OEP
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Once the Authors have asked the Open edX community whether an idea has any chance
 of acceptance, a draft OEP should be submitted as a pull request against the
-`central OEP repository`_. To identify the draft proposal, the Author should
+`central OEP repository`_. To identify the draft proposal, the Authors should
 check the numbered list of previous OEP pull requests and select the next 
-available number. The pull request title must start with "OEP-XXX", which 
-claims that OEP number for the included proposal.
+available number. The pull request title should be of the form 
+"OEP-XXXX-YYYY-ZZZZ", where
```

Comment:
It seems like the title of the PR should be human-centric, and that maybe we should use GH tags for things like process, best practice, or architecture.

Reply:
I see your point.  Do you think bp-proc-arch aren't a quick human-centric way to read the type of the OEP in the title (for developers)?

Reply:
I'd much rather have real text in the title, and have tags with real words as well.

Something like "OEP-1: A Process for Enhancing OpenEdX", tagged "process"

Reply:
That sounds good.

Reply:
updated.